electron/src/services/uhk-device-provider.ts

```diff
@@ -0,0 +1,18 @@
+import { Provider } from '@angular/core';
+
+import { UhkDeviceService } from './uhk-device.service';
+import { UhkHidApiService } from './uhk-hid-api.service';
+
+export function uhkDeviceProvider(): Provider {
+    // HID API officially support MAC, WIN and linux x64 platform
+    // https://github.com/node-hid/node-hid#platform-support
+    if (process.platform === 'darwin' ||
```

Comment:
Should I do anything until we will finish the test on all platform?
Currently the app use UhkHidApiService on all platform.
If we drop UhkLibUsbApiService from the code, then I like to remove the uhkDeviceProvider and use directly UhkHidApiService every where.